src/gatsby/models/docs/onCreateMarkdownContentNode.js

```diff
@@ -10,10 +10,7 @@ function createMarkdownDocsNode(api, { parentNode }) {
   splitDir[0] = 'doc'
 
   // Make a special exemption for the root doc.
```

Comment:
is it still a special "exemption"? or a general rule now?

Reply:
That's just a comment I forgot, good catch.